.env

```diff
@@ -0,0 +1,5 @@
+# version for opensearch & opensearch-dashboards docker image
+VERSION=2.6.0
+# auth details - change according to your system
+USER=admin
+PASSWORD=admin
```

Comment:
Can we replace `.env` file with `env-example` and ask user to explicitly change this before using in the `Security Notice` section in using-docker section.